test/integration/scheduler/framework_test.go

```diff
@@ -53,6 +53,13 @@ type PrefilterPlugin struct {
 	TesterPlugin
 }
 
+type ScorePlugin struct {
+	TesterPlugin
```

Comment:
We don't have a `TesterPlugin` in the master branch, please take a look

Reply:
the PR needed rebasing.